cypress/integration/articleSpec.js

```diff
@@ -89,4 +89,22 @@ describe('News Article', () => {
       .should('have.attr', 'src')
       .and('match', /(\/static\/js\/bundle\.\w+\.js)/g);
   });
+
+  it('should check the font file loads', () => {
+    const fontFamiliesArray = [];
+    cy.get('*')
```

Comment:
This is very inclusive and includes Cypress elements. This is why `Times` is being returned as a font. We should just get a parent element that is specific to our content.